pkg/apis/storage/validation/validation_test.go

```diff
@@ -1769,6 +1770,14 @@ func TestCSIDriverValidation(t *testing.T) {
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: driverName},
```

Comment:
Test case for invalid string?

Reply:
I did this in two steps:

>	invalidFSGroupPolicy := storage.HeuristicFSGroupPolicy
>	invalidFSGroupPolicy = "invalid-mode"

But there is now a test for an invalid mode, which fails as expected.